### ISSUE TITLE
Printer without ethernet shield

### ISSUE THREAD

**Issue by lnw222** [exciting-io/printer#49]
Can I use the code for the different prints without having the ethernet shield? I'm new to Arduino and coding so I'm having a bit of trouble. I plan on keeping the printer connected to my computer with the USB but I want to print the things available here using the code for the shield. How hard would it be to modify it for that purpose?

**Comment by lazyatom** [comment]
Using the Arduino without the ethernet shield would change the way you interact with the service quite a bit. Since the Arduino would simply be acting as a conduit between the printer and your computer, I think you wouldn't actually use the `printer.ino` sketch at all; instead, you'd probably want to write a small bit of software to poll the server API on your computer, and then send any returned data to the printer via the Arduino.

So, in short, it's certainly possible, but it'd require a bit of software development on your part. Perhaps a useful analogue is [the software others have written to connect using a Raspberry Pi](https://github.com/andrewn/raspi-printer/blob/master/printer.py), since that is, in effect, more like a computer than an Arduino. There's no reason why that same software cannot run on your computer...

**Comment by jems-ford** [comment]
A printer is used to the printing scanning the documents. You can simply be used the Ethernet cable using the RJ 45 Connector for connecting the printer. most of the software is available for the printer using for enternet shield. If your HP printer is not working in the LAN please contact [Hp Printer Support Phone Number](http://www.printererrorrepair.com/hp-printer-support-phone-number/) for the instant solution.
